source/guides/technology-documentation-sites/index.html.md.erb

```diff
@@ -61,7 +61,7 @@ All the workspace-related settings (workspace ID, API key and API secret) can be
 
 Next we must setup the GitHub Actions to publish:
 
-3. Copy the [automation script](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`.
+3. Copy the automation script from [here](https://github.com/DFE-Digital/academy-transfers-technical-documentation/blob/master/source-c4-diagrams/export-private-cloud-diagrams.js) to `<repository-root>/source-c4-diagrams/export-private-cloud-diagrams.js`.
```

Comment:
"here" is not ideal for accessibility. the link title should represent the content.

Reply:
Ah yes you're right. I must have done something weird with my branch as I'm sure we fixed that. Thanks for the spot